src/Fgsl/Eyedatagrid/EyeDataGrid.php

```diff
@@ -511,17 +511,18 @@ private function buildHeader()
 
             // Add filters if allowed and only if the column type is not "special"
             if ($this->allow_filters and
-                !in_array($this->type[$t][0], array(
-                                    self::TYPE_ARRAY,
-                                    self::TYPE_IMAGE,
-                                    self::TYPE_FUNCTION,
-                                    self::TYPE_DATE,
-                                    self::TYPE_CHECK,
-                                    self::TYPE_CUSTOM,
-                                    self::TYPE_PERCENT
-                                    )))
+                (!isset($this->type[$t]) or

```

Comment:
Additional check if the column type is specified. In case it wasn't we're not dealing with special case, so we can use filters.